packages/amazonq/.changes/next-release/Feature-b771fe9a-7f29-47b0-b39c-1282224069dd.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "feat(amazonq): enable code actions for awesome lightbulb"
```

Comment:
Thanks for including a changelog. Please review the [changelog guidelines](https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog). Is this the most meaningful thing we can say about this improvement for customers that will be interested in it?